tfhe/benches/high_level_api/erc20.rs

```diff
@@ -263,26 +275,106 @@ fn bench_transfer_throughput<FheType, F>(
         );
     }
 }
+#[cfg(feature = "gpu")]
+fn cuda_bench_transfer_throughput<FheType, F>(
+    group: &mut BenchmarkGroup<'_, WallTime>,
+    client_key: &ClientKey,
+    bench_name: &str,
+    type_name: &str,
+    fn_name: &str,
+    transfer_func: F,
+) where
+    FheType: FheEncrypt<u64, ClientKey> + Send + Sync,
+    F: for<'a> Fn(&'a FheType, &'a FheType, &'a FheType) -> (FheType, FheType) + Sync,
+{
+    let mut rng = thread_rng();
+    let num_gpus = get_number_of_gpus() as u64;
+    let compressed_server_key = CompressedServerKey::new(client_key);
+
+    let sks_vec = (0..num_gpus)
+        .map(|i| compressed_server_key.decompress_to_specific_gpu(GpuIndex::new(i as u32)))
+        .collect::<Vec<_>>();
+
+    for num_elems in [10 * num_gpus, 100 * num_gpus, 500 * num_gpus] {
+        group.throughput(Throughput::Elements(num_elems));
+        let bench_id = format!("{bench_name}::{fn_name}::{type_name}::{num_elems}_elems");
```

Comment:
Could you add a `::throughput` after `{bench_name}` ?
Otherwise we cannot disambiguate latency VS throughput cases based on benchmark name.